lib/util/fonts.ts

```diff
@@ -0,0 +1,36 @@
+//font optimization from @next/font
+import { Inter } from "@next/font/google";
+import { Kanit } from "@next/font/google";
+import { Roboto_Mono } from "@next/font/google";
+
+export const inter = Inter({
+  subsets: [
```

Comment:
This one doesn’t have weights specified, does it load all by default?

Reply:
Yes, normally you do need to specify a range with variable fonts but because it is self hosted by nextJS it loads the range by default.